### ISSUE TITLE
No Video at 1080P for cores built with the latest Template_MiSTer.

### ISSUE THREAD

**Issue by MiskatonicU** [MiSTer-devel/Template_MiSTer#61]
**TLDR**

It appears that some recent change in the MiSTer Template makes 1080P through my A/V Receiver to my TV no longer work, though 720P continues to work.  When hooked directly to the TV, 1080P works.  Older builds of cores, including the PSX one, work at both 720P and 1080P.  The receiver is older, but previously had no issues.  It's a Yamaha RX-V667.

I believe the issue may be due to this change: 
https://github.com/MiSTer-devel/PSX_MiSTer/commit/5aeb43411882470884c4535fc03768c2cd76bf86

Which I believe was part of this issue:
https://github.com/MiSTer-devel/Template_MiSTer/pull/59

**Full Issue Description**

I've been using the PSX core betas for months without issue, running at 1080P with vsync_adjust set to 2, though also using 0 on occasion for some games that switch resolution frequently.

Since the PSX core was recently released I updated via the 1.4 Downloader to the latest MiSTer Main and the released PSX core, both dated 5/11/2022.  I've also verified the SHA hashes of both to confirm they weren't corrupted. I'm also using a CEC-Less cable out of the MiSTer to the input.

After updating I can no longer get 1080P with any vsync_adjust value option, though 720P continues to work no matter the vsync_adjust used.  The most recent PSX core test build from 5/9/2022 also continues to work fine at 1080P on the same MiSTer Main.  In this case 'No Video' means the TV does not see a signal at all, it doesn't even attempt to sync.

Upon advice in the #debug channel on the discord I tried another recent core that would have the same change as the official PSX release from 5/11/2022, this being the Jackal Arcade core.  It also exhibits the same issue, with 1080P output being broken, but 720P output being fine.

After some further recommendations from the #debug chat, I tried with a different TV, where 1080P worked as expected.  I then tried plugging the MiSTer directly into the problem TV, rather than through my A/V setup, and surprisingly 1080P also worked there.  However, it stopped working again once I plugged MiSTer back into my A/V Receiver,  a Yamaha RX-V667.

So it seems like some recent change in the template has broken 1080P when routed through my receiver to my TV.  With my current A/V Setup having the MiSTer plugged directly into the TV isn't ideal, and I would prefer to continue routing it through my Yamaha RX-V667, but I do understand this may be an edge case and I'll just have to redo my A/V setup to accommodate having the MiSTer attached directly to the TV.

**Comment by birdybro** [comment]
The latest Main MiSTer binary (unstable build) has a change to default game mode to 0, but the template defaults to 1 for users. If a new binary is pushed out soon it should remedy any of these issues.

**Comment by MiskatonicU** [comment]
@wickerwaka In the #debug channel on Discord built a new MiSTer Main which seems to resolve this issue for me.  His notes:

"My change just re-enables VIC auto-detection."

Would it be possible to have a config flag to toggle this behavior rather than removing it entirely as the most recent change seems to indicate is the case?

**Comment by birdybro** [comment]
It should probably be enabled by default to maintain current compatibility, and something that users could disable in the .ini placed just below the other hdmi_config option that was added recently.

**Comment by MiskatonicU** [comment]
I tested an updated build of Main that @wickerwaka provided and can confirm the fix he has suggested resolves the issues I was seeing.
